.golangci.yml

```diff
@@ -52,7 +50,6 @@ linters:
     - godox
     - gofmt
     - goimports
-    - golint
```

Comment:
`lint` [was deprecated by golang developers](https://github.com/golang/go/issues/38968)

Reply:
`revive` is an alternative used now